milestone2.ts

```diff
@@ -83,34 +85,65 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) => {
+  let total = 0;
+  for (var volunteer of data) {
+    total += volunteer.age;
+  }
+  return total / data.length;
+};
 
-//console.log(findAverage(userData));
+// console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data) => {
+  let count = 0;
+  for (var volunteer of data) {
+    if (volunteer.city == "SF") {
+      return count;
+    }
+    else {
+      count += 1;
+    }
+  }
+  return -1;
```

Comment:
For questions 2-4 please use the data handling functions shown [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array). For loops are good, but its important to be familiar with lambda function syntax too. Sorry if the instructions were unclear!